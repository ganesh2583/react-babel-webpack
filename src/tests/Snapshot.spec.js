import React from "react";
import renderer from "react-test-renderer";

import RootApp from "./../index";

describe("Test Counter", () => {
  it("Test onClick increase the count", () => {
    const component = renderer.create(<RootApp />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
