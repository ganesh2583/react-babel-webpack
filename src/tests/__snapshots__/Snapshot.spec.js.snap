// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Counter Test onClick increase the count 1`] = `
<div
  className="home-page-container"
>
  <div
    className="home-page-item"
  >
    React with Webpack and Babel
  </div>
  <div
    className="home-page-item"
  >
    Im root
    <div
      id="counterId"
    >
      Counter = 
      0
      <button
        id="incrementBtn"
        onClick={[Function]}
      >
        Click to increment
      </button>
    </div>
  </div>
</div>
`;
